types/global.d.ts

```
@@ -222,6 +230,46 @@ declare global {
+    [key: string]: any; // Allow passthrough of any other fields Finnhub may return
+  };
+
+  type GetWatchlistPageParams = {
+    page?: number; // 1-based
+    pageSize?: number;
+    search?: string; // search by symbol/company
+    filter?: "all" | "gainers" | "losers" | "favorites";
+    sort?: string; // e.g. "symbol:asc" or "addedAt:desc"
+  };
+
+  type WatchlistPageResult = {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'WatchlistPageResult'. Consider to delete it or rename it.

'WatchlistPageResult' is defined here:


(lint/suspicious/noRedeclare)